packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoinifyCurrency/index.js

```diff
@@ -12,6 +12,28 @@ const CustomSelectBox = styled(SelectBox)`
   * {
     border-left: none;
   }
+  .bc__control {
+    border: 1px solid ${props => props.theme['gray-2']};
+    border-left: none;
+    > .bc__value-container {
+      > .bc__single-value {
+        right: 0px;
+      }
+    }
+    > .bc__indicators {
+      > .bc__indicator-separator {
+        display: none;
+      }
+    }
+    &:hover {
+      border: 1px solid ${props => props.theme['gray-2']};
+      border-left: none;
+    }
+    &:active {
+      border: 1px solid ${props => props.theme['gray-2']};
+      border-left: none;
+    }
+  }
```

Comment:
Please also target bc__single-value and make sure the font-size and color is the same as the other field's coin type

Reply:
Woops, sorry about that. Should be fixed now for the selected currency and items in the dropdown.